sonar-slang-plugin/src/main/resources/org/sonar/l10n/kotlin/rules/kotlin/S1192.json

```diff
@@ -0,0 +1,18 @@
+{
+  "title": "String literals should not be duplicated",
+  "type": "CODE_SMELL",
+  "status": "ready",
+  "remediation": {
+    "func": "Linear with offset",
+    "linearDesc": "per duplicate instance",
+    "linearOffset": "2min",
+    "linearFactor": "2min"
```

Comment:
That's not consistent with the issues we create: we should report a "gap" which will be used to compute the issue debt.